duckduckgo/iOS
# add open in DuckDuckGo action

Task/Issue URL: https://app.asana.com/0/392891325557410/1172824025123186
Tech Design URL:
CC:

**Description**:

Adds an open action.

**Steps to test this PR**:
1.  Share a URL from another app.  
1.  "Open in DuckDuckGo" should appear in the list of actions on the sheet.
1. Tapping Open in DuckDuckGo should launch the URL in a new tab in the DuckDuckGo app

Note that before iOS 13 you will need to enable this, but from iOS 13 you don't.


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
